tools/train.py

```diff
@@ -67,7 +66,7 @@ def main():
 
     model = build_detector(
         cfg.model, train_cfg=cfg.train_cfg, test_cfg=cfg.test_cfg)
-    train_dataset = obj_from_dict(cfg.data.train, datasets)
+    train_dataset = datasets.get_dataset(cfg.data.train)
```

Comment:
Since we just need to use `get_dataset()` from the `datasets` module, I recommend `from mmdet.datasets import datasets`.

Reply:
@hellock Do you mean `from mmdet.datasets import get_dataset` ?